ui/app/templates/components/wizard/replication-setup.hbs

```diff
@@ -11,15 +11,9 @@
       you more interested in speed of access?
     </p>
 
-    <DocLink @path="/vault/tutorials/enterprise/performance-replication">
```

Comment:
As an FYI, OpenBao likely will not have either mode as they're not on our roadmap. We do have HA mode though, so in the future, this doc page will likely have to be reworked.